src/TraceEvent/Parsers/ClrPrivateTraceEventParser.cs

```diff
@@ -1397,19 +1397,32 @@ public enum Keywords : long
                 source.UnregisterEventTemplate(value, 39, GCTaskGuid);
             }
         }
-        public event Action<CCWRefCountChangeTraceData> GCCCWRefCountChange
+        public event Action<CCWRefCountChangeAnsiTraceData> GCCCWRefCountChangeAnsi
         {
             add
             {
                 // action, eventid, taskid, taskName, taskGuid, opcode, opcodeName, providerGuid, providerName
-                RegisterTemplate(new CCWRefCountChangeTraceData(value, 193, 1, "GC", GCTaskGuid, 40, "CCWRefCountChange", ProviderGuid, ProviderName));
+                RegisterTemplate(new CCWRefCountChangeAnsiTraceData(value, 193, 1, "GC", GCTaskGuid, 40, "CCWRefCountChange", ProviderGuid, ProviderName));
             }
             remove
             {
                 source.UnregisterEventTemplate(value, 193, ProviderGuid);
                 source.UnregisterEventTemplate(value, 40, GCTaskGuid);
             }
         }
+        public event Action<CCWRefCountChangeTraceData> GCCCWRefCountChange
+        {
+            add
+            {
+                // action, eventid, taskid, taskName, taskGuid, opcode, opcodeName, providerGuid, providerName
+                RegisterTemplate(new CCWRefCountChangeTraceData(value, 200, 1, "GC", GCTaskGuid, 40, "CCWRefCountChange", ProviderGuid, ProviderName));
+            }
+            remove
+            {
+                source.UnregisterEventTemplate(value, 200, ProviderGuid);
+                source.UnregisterEventTemplate(value, 40, GCTaskGuid);
```

Comment:
Remove the line that removes the opcode 40 GCTaskGuid.   Arguably, it should not even be there for the other version of the event (since it is for pre-Vista compat, and this event does not care about that compat).    However if it is in, you have to have a unique opcode-Task pair, or weird things will happen.     Suffice it to say that less is better.